Mathlib/CategoryTheory/Limits/Shapes/Types.lean

```diff
@@ -711,12 +711,206 @@ end Pullback
 
 section Pushout
 
--- #synth HasPushouts.{u} (Type u)
-instance : HasPushouts.{u} (Type u) :=
-  -- FIXME does not work via `inferInstance` despite `#synth HasPushouts.{u} (Type u)` succeeding.
-  -- https://github.com/leanprover-community/mathlib4/issues/5752
-  -- inferInstance
-  hasPushouts_of_hasWidePushouts.{u} (Type u)
```

Comment:
Does `inferInstance` work now? If yes, then please mention in the PR description.

Reply:
Yes, it does work now.

Reply:
(But only for pushouts, there is still an issue with pullbacks.)